### ISSUE TITLE
Items being miscategorized

### ISSUE THREAD

**Issue by benphantom41** [Haselnussbomber/LeveHelper#8]
Some items are being miscategorized like Copper ore, Titanium Ore, and Mythrite Ore
![image](https://github.com/Haselnussbomber/LeveHelper/assets/48699776/27a6ab42-755a-4f49-9c3f-b112373019e7)

When clicking on one of them, all the current map icons are removed
![image](https://github.com/Haselnussbomber/LeveHelper/assets/48699776/14a89fcf-529f-4306-8e18-3a60013afa1d)

I do have every single Shadowbringers Blacksmith Levequest accepted, just thought I would mention it

**Comment by Haselnussbomber** [comment]
Yeah, that's what I expected when I removed the territory check last update.

I noticed that even GatherBuddy has the hidden items hardcoded, that you mentioned in issue #7 ([here](https://github.com/Ottermandias/GatherBuddy/blob/main/GatherBuddy.GameData/Data/HiddenItems.cs#L12-L36)).
Maybe I can find some more info from TeamCraft or GarlandTools, too, or just make use of that GatherBuddy list.

I'll check it out soon.
